src/internal/m365/collection/drive/limiter_test.go

```diff
@@ -664,118 +525,45 @@ func (suite *LimiterUnitSuite) TestGet_PreviewLimits() {
 			},
 		},
 	}
-	for _, test := range table {
-		suite.Run(test.name, func() {
-			t := suite.T()
+}
+
+// TestGet_PreviewLimits checks that the limits set for preview backups in
+// control.Options.ItemLimits are respected. These tests run a reduced set of
+// checks that don't examine metadata, collection states, etc. They really just
+// check the expected items appear.
+func (suite *LimiterUnitSuite) TestGet_PreviewLimits_noTree() {
+	metadataPath, err := path.BuildMetadata(
+		tenant,
+		user,
+		path.OneDriveService,
+		path.FilesCategory,
+		false)
+	require.NoError(suite.T(), err, "making metadata path", clues.ToCore(err))
```

Comment:
nit: could you please move this to `runGetPreviewLimits` since it's not used in this function beyond that? The same will likely apply to making the drives, depending on whether they're actually passed into the function that generates the test cases or not

Reply:
done